build/source/engine/systemSolv.f90

```diff
@@ -1487,68 +1290,35 @@ subroutine xFluxResid(&
-   !if(printFlag)then
-   ! print*, 'rAdd(ixVegNrg), scalarCanopyIceLocal, scalarCanopyIce = ', rAdd(ixVegNrg), scalarCanopyIceLocal, scalarCanopyIce
-   ! pause
-   !endif
   endif
 
   ! compute energy associated with melt/freeze for snow
   if(nSnow>0)&
   rAdd(ixSnowOnlyNrg) = rAdd(ixSnowOnlyNrg) + LH_fus*iden_ice*(mLayerVolFracIceLocal(1:nSnow) - mLayerVolFracIce(1:nSnow))       ! energy associated with melt/freeze (J m-3)
-  if(printFlag)then
-   !write(*,'(a,1x,10(e20.10,1x))') 'rAdd(ixSnowOnlyNrg) = ', rAdd(ixSnowOnlyNrg)
-   !write(*,'(a,1x,10(e20.10,1x))') 'mLayerVolFracIce(1:5) = ', mLayerVolFracIce(1:5)
-   !write(*,'(a,1x,10(e20.10,1x))') 'mLayerVolFracIceLocal(1:5) = ', mLayerVolFracIceLocal(1:5)
-   !write(*,'(a,1x,10(e20.10,1x))') 'delIce = ', mLayerVolFracIceLocal(1:5) - mLayerVolFracIce(1:5)
-  endif
 
   ! compute energy associated with melt/freeze for soil
   rAdd(ixSoilOnlyNrg) = rAdd(ixSoilOnlyNrg) + LH_fus*iden_water*(mLayerVolFracIceLocal(nSnow+1:nLayers) - mLayerVolFracIce(nSnow+1:nLayers))     ! energy associated with melt/freeze (J m-3)
-  !if(printFlag)then
-  ! write(*,'(a,1x,10(e30.15,1x))') 'vIce01 = ', mLayerVolFracIce(nSnow+1:nLayers)
-  ! write(*,'(a,1x,10(e30.15,1x))') 'vIce02 = ', mLayerVolFracIceLocal(nSnow+1:nLayers)
-  ! write(*,'(a,1x,10(e15.5,1x))') 'delIce = ', mLayerVolFracIceLocal(nSnow+1:nLayers) - mLayerVolFracIce(nSnow+1:nLayers)
-  ! write(*,'(a,1x,10(e15.5,1x))') 'delNrg = ', LH_fus*iden_water*(mLayerVolFracIceLocal(nSnow+1:nLayers) - mLayerVolFracIce(nSnow+1:nLayers))
-  ! print*, 'precision(mLayerVolFracIce) = ', precision(mLayerVolFracIce)
-  !endif
 
   ! sink terms (-)
   ! NOTE: state variable is volumetric water content, so melt-freeze is not included
   ! NOTE: ground evaporation was already included in the flux at the upper boundary
-  !print*, 'mLayerTranspire(1:nSoil) = ', mLayerTranspire(1:nSoil)
-  !print*, 'mLayerBaseflow(1:nSoil)  = ', mLayerBaseflow(1:nSoil)
-  !print*, 'mLayerCompress(1:nSoil)  = ', mLayerCompress(1:nSoil)
-
-  !mLayerCompress(:)  = 0._dp
-  !mLayerTranspire(:) = 0._dp
-  !mLayerBaseflow(:) = 0._dp
   rAdd(ixSoilOnlyMat)    = rAdd(ixSoilOnlyMat) + dt*(mLayerTranspire(1:nSoil) - mLayerBaseflow(1:nSoil) )/mLayerDepth(nSnow+1:nLayers) - mLayerCompress(1:nSoil)
-  !print*, 'rAdd(ixSoilOnlyMat)      = ', rAdd(ixSoilOnlyMat)
 
   ! liquid water equivalent of melt/freeze for snow layers (-)
   ! NOTE: state equation for soil is based on the total equivalent liquid water content (liquid plus ice)
-  !if(nSnow>0)&
-  !rAdd(ixSnowOnlyWat) = rAdd(ixSnowOnlyWat) - (iden_ice/iden_water)*(mLayerVolFracIceLocal(1:nSnow) - mLayerVolFracIce(1:nSnow)) ! liquid water equivalent of melt/freeze (-)
-  !if(printFlag)then
-  ! write(*,'(a,1x,10(e20.10,1x))') 'rAdd(ixSnowOnlyWat) = ', rAdd(ixSnowOnlyWat)
-  !endif
+  if(nSnow>0)&
+  rAdd(ixSnowOnlyWat) = rAdd(ixSnowOnlyWat) ! included for clarity 
```

Comment:
Not clear how this is supporting "clarity", please explain in comment what it actually does, because "as is", I am not sure what this does. This is probably my ignorance speaking, but why `&` after the `if` statement and shouldn't the `rAdd` line be indented?